component/main.jsonnet

```diff
@@ -45,6 +45,23 @@ local sub =
       subscription.name,
       subscription.channel
     ) + {
+      spec+: {
```

Comment:
Looking at it again, it would probably make more sense to add the default node selector config in the component library, so that other components which deploy subscriptions benefit from the new default as well.

Reply:
That makes a lot more sense 👍